### ISSUE TITLE
Disable number separator

### ISSUE THREAD

**Issue by JonnyTech** [EvanRespaut/Equate#27]
Is it possible to disable number separators? I prefer to see `1234567` rather than `1,234,567`.

**Comment by JonnyTech** [comment]
Separators were added with commit https://github.com/EvanRespaut/Equate/commit/827bfae78b610d0e8a5e0b39e894ef9cdcdc313b but I do not know how to make that into an optional setting.

**Comment by EvanRespaut** [comment]
Hey sorry, I've been crazy busy at work, not a lot of time left for side projects. I'll try to get to making some updates soon though, this one should be pretty straight forward. 

If you wanted to make that into an optional setting, you'd need to edit the Settings Activity. You'd also need to change preferences.xlm file in the res directory.

**Comment by JonnyTech** [comment]
Thanks for the reply. I have been trying to work out how Android development works but not able to get a successful compile yet. Looking forward to your update whenever you get a chance in the future.

**Comment by EvanRespaut** [comment]
Yeah Android development is not always straightforward. What kind of errors your getting on compile?

**Comment by JonnyTech** [comment]
I think that it is my installation, lots on the go and I don't wanna fix my distro for fear of breaking something else. Will do so when current projects done. Thanks for the offer.
